packages/console/src/components/AppContent/components/Sidebar/components/Item/index.tsx

```diff
@@ -29,28 +30,36 @@ const Item = ({ icon, titleKey, modal, isActive = false }: Props) => {
     [icon, t, titleKey]
   );
 
-  if (!modal) {
+  if (modal) {
     return (
-      <Link to={getPath(titleKey)} className={classNames(styles.row, isActive && styles.active)}>
+      <>
+        <button
+          className={styles.row}
+          onClick={() => {
+            setIsOpen(true);
+          }}
+        >
+          {content}
+        </button>
+        {modal(isOpen, () => {
+          setIsOpen(false);
+        })}
+      </>
+    );
+  }
+
+  if (externalLink) {
+    return (
+      <a href={externalLink} target="_blank" rel="noreferrer" className={styles.row}>
```

Comment:
Remove `noreferrer` because we would like to know the referrer from our docs site.
But we can use `noopener` here to improve security